Deployment/Upgrade Service Fabric cluster basic load balancer.md

```diff
@@ -0,0 +1,444 @@
+# Upgrade Service Fabric cluster from Basic load balancer to Standard load balancer SKU
+
+## Overview  
+
+This documents the options available to upgrade a basic load balancer sku to standard load balancer sku for a Service Fabric cluster. Choose one of the options below based on availability requirements.
+
+### NOTE: This does not apply to [Service Fabric managed clusters](https://learn.microsoft.com/azure/service-fabric/overview-managed-cluster). Service Fabric managed clusters with 'Basic' SKU are provisioned with a 'Basic' load balancer but cannot be upgraded and must be redeployed . Service Fabric managed clusters with 'Standard' SKU have are provisioned with a 'Standard' load balancer and are not impacted.
+
+## Manual Upgrade Process with no down time
+
+To upgrade basic load balancers in a Service Fabric cluster with no downtime requires the creation of a new scale set (node type), standard load balancer, and standard IP address. The new node type is added to the cluster, applications/services are migrated to new node type, old node type with associated basic load balancer and IP address are deactivated and removed. This process takes multiple hours to complete and is documented in [Scale up a Service Fabric cluster primary node type](https://learn.microsoft.com/azure/service-fabric/service-fabric-scale-up-primary-node-type) and [Scale up a Service Fabric cluster non-primary node type](https://learn.microsoft.com/azure/service-fabric/service-fabric-scale-up-non-primary-node-type).
```

Comment:
mention needing to also add NSG w/Standard LB here.

Please open PR on primary sku upgrade doc and mention NSG required for Std LB